happy_bank_core/logic/account.py

```diff
@@ -0,0 +1,14 @@
+accounts = dict()
+accounts["#1234"] = {"name": "John Doe", "deposit": 1000}
+accounts["#12345"] = {"name": "Johanna Doe", "deposit": 2000}
+
+
+class Account:
+    def __init__(self, customer):
+        """Gets customer's data from accounts dictionary"""
+        self.id = customer
+        self.name = accounts[self.id]["name"]
+        self.deposit = float(accounts[self.id]["deposit"])
```

Comment:
Account should be such a data class without parsing logic in init method. If we need to parse json, let's create it afterward.

Reply:
So we do not need accounts dictionary at the moment and we are going to pass all arguments manually to Account class, right?

Reply:
If you mean manually just like this, than yes:
```suggestion
        self.id = id
        self.full_name = full_name
        self.balance = balance
```